tensorflow/python/tools/saved_model_cli.py

```diff
@@ -949,11 +947,6 @@ def create_parser():
       'tensorrt',
       description='Convert the SavedModel with Tensorflow-TensorRT integration',
       formatter_class=argparse.RawTextHelpFormatter)
-  parser_convert_with_tensorrt.add_argument(
-      '--max_batch_size',
-      type=int,
-      default=1,
-      help='max size for the input batch')
```

Comment:
Are we replacing this argument with something else?  Or is it not needed anymore?

Reply:
My understanding is that `max_batch_size` was removed in the implementation of TensorRT for TensorFlow 2.0 and not replaced with anything.